pkgs/development/python-modules/typedload/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  attrs,
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  pytestCheckHook,
+  python,
+  setuptools,
+}:
+
+let
+  typedload = buildPythonPackage rec {
+    pname = "typedload";
+    version = "2.37";
+    format = "pyproject";
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "DNYPufshGeAGQ7wwLs5nANoxRtFFpsRaOeLsgVHXQLE=";
```

Comment:
Can you use `hash = "sha256-xxx"`? The `sha256 = ...` form is deprecated
- #325892

Reply:
sorry this comment was for the next item :))) Yep, fixed in 0f9707085d655e1fc51539fb1f0264e5ac48767d; thanks!